src/XCCDF/xccdf_session.c

```diff
@@ -414,6 +416,13 @@ bool xccdf_session_set_xccdf_export(struct xccdf_session *session, const char *x
 	return true;
 }
 
+bool xccdf_session_set_stigviewer_export(struct xccdf_session *session, const char *stig_rule_id_file)
```

Comment:
stig_rule_id_file sounds like it's a file of stig rule IDs where it's actually an XCCDF results file